numberFormatter.js

```diff
@@ -52,84 +52,47 @@ define(function () {
         }
     };
 
-    var serviceSeperatorFormats = {
-        english:           commonSeperatorFormats.commasAndDecimals,
-        arabic:            customSeperatorFormats.arabic,
-        azeri:             commonSeperatorFormats.spacesAndCommas,
-        bengali:           commonSeperatorFormats.commasAndDecimals,
-        brasil:            commonSeperatorFormats.spacesAndCommas,
-        burmese:           commonSeperatorFormats.commasAndDecimals,
-        chinese:           commonSeperatorFormats.commasAndDecimals,
-        chinese_simp:      commonSeperatorFormats.commasAndDecimals,
-        chinese_trad:      commonSeperatorFormats.commasAndDecimals,
-        ukchina_simp:      commonSeperatorFormats.commasAndDecimals,
-        ukchina_trad:      commonSeperatorFormats.commasAndDecimals,
-        french:            commonSeperatorFormats.spacesAndCommas,
-        gahuza:            commonSeperatorFormats.decimalsAndCommas,
-        hausa:             commonSeperatorFormats.commasAndDecimals,
-        hindi:             customSeperatorFormats.hindi,
-        indonesia:         commonSeperatorFormats.decimalsAndCommas,
-        indonesian:        commonSeperatorFormats.decimalsAndCommas,
-        kyrgyz:            commonSeperatorFormats.spacesAndCommas,
-        mundo:             commonSeperatorFormats.decimalsAndCommas,
-        news:              commonSeperatorFormats.commasAndDecimals,
-        nepali:            commonSeperatorFormats.commasAndDecimals,
-        persian:           commonSeperatorFormats.commasAndDecimals,
-        pashto:            commonSeperatorFormats.commasAndDecimals,
-        portuguese:        commonSeperatorFormats.decimalsAndCommas,
-        russian:           commonSeperatorFormats.spacesAndCommas,
-        sinhala:           commonSeperatorFormats.commasAndDecimals,
-        sinhali:           commonSeperatorFormats.commasAndDecimals,
-        somali:            commonSeperatorFormats.commasAndDecimals,
-        spanish:           commonSeperatorFormats.decimalsAndCommas,
-        swahili:           commonSeperatorFormats.commasAndDecimals,
-        tamil:             commonSeperatorFormats.commasAndDecimals,
-        thai:              commonSeperatorFormats.commasAndDecimals,
-        turkish:           commonSeperatorFormats.decimalsAndCommas,
-        urdu:              commonSeperatorFormats.commasAndDecimals,
-        ukrainian:         commonSeperatorFormats.spacesAndCommas,
-        uzbek:             commonSeperatorFormats.spacesAndCommas,
-        vietnamese:        commonSeperatorFormats.decimalsAndCommas
+    var serviceSeperatorFormats = function(service){
+        switch(service){
```

Comment:
Could we keep the dictionary pattern of before, but combine `serviceSeperatorFormats` and `percentageFormatIsAppend`:

```js
const serviceSeperatorFormats = {
  english: {
    separators: types.COMMAS_AND_DECIMALS,
    appendPercentage: true,
  },
  // etc
}
```